working/wildcards/feature-specification.md

```diff
@@ -127,8 +135,31 @@ means you can have multiple local declarations named `_` in the same namespace
 without a collision error. The initializer, if there is one, is still executed,
 but the value is not accessible.
 
-Other declarations: top-level variables, top-level function names, type names,
-member names, etc. are unchanged. They can be named `_` as they are today.
+### Record type positional fields
+
+```dart
+typedef R = (String _, String _);
+(int _, int _) record;
+```
+
+Record type field names will no longer emit an `INVALID_FIELD_NAME` warning.
```

Comment:
I believe it's [an error](https://github.com/dart-lang/language/blob/main/accepted/3.0/records/feature-specification.md#record-expressions), not a warning, for a record field name to start with `_`.

Also, the language specifies when an error is reported, but not which error. Here, I'd probably say something like:

> It is currently an error for a record field name to begin with `_` (including just a bare `_`). We relax that error to only apply to record fields whose name begins with `_` followed by at least one other character (even if those later character(s) are `_`).

Reply:
Argh, words. Thank you. I'll just steal that :)

> Also, the language specifies when an error is reported, but not which error.

And this is also good to know.

Reply:
And it's still a compile-time error for a *named* field's name to start with `_`, it's only positional field names that can be ignored.

Reply:
Added an extra line on named fields. I think it's good to reiterate that. Thanks.